mint.json

```diff
@@ -70,7 +68,7 @@
       "pages": [
         "resources/overview",
         {
-          "group": "Databases",
+          "group": "Data Persistence",
```

Comment:
WDYT? @ivancernja

Reply:
Can revert this, if not desirable, it was just a thought.

Reply:
@jonaro00 @sentinel1909 I'm not a fan of it, to be honest. I feel like "Databases" would resonate better with both beginners and experienced developers whereas "Data Persistance" might put some beginners off. Plus, if we look at other platforms, iirc they all use the term "Databases".

Reply:
Understood, fixed. Name changed back to "Databases"